src/Netpbm.jl

```diff
@@ -150,19 +150,17 @@ function parse_netpbm_size(stream::IO)
 end
 
 function parse_netpbm_maxval(stream::IO)
-    skipchars(stream, isspace, linecomment='#')
+    skipchars(isspace, stream, linecomment='#')
     maxvalline = strip(readline(stream))
     parse(Int, maxvalline)
 end
 
 function parseints(line, n)
-    ret = Vector{Int}(n)
+    ret = Vector{Int}(undef, n)
     pos = 1
     for i = 1:n
-        pos2 = search(line, ' ', pos)
-        if pos2 == 0
-            pos2 = length(line)+1
-        end
+        space = findnext(" ", line, pos)
+        pos2 = space === nothing ? length(line)+1 : space[1]
```

Comment:
Do you need the `[1]` after `space`?

Reply:
Yes: findnext now returns either nothing or a UnitRange (https://docs.julialang.org/en/v1/base/strings/#Base.findnext-Tuple{AbstractString,AbstractString,Integer}), so the first index of that range is space[1]

Reply:
Ah, I hadn't caught the `" "` (string rather than char). Try this:
```julia
julia> findnext(isequal(' '), "hello and good day", 1)
6

julia> findnext(isequal(' '), "hello and good day", 7)
10
```

Reply:
I see, I scrolled through Base functions and found the wrong replacement for search.

Reply:
I think the `x = f(); x = (x === nothing ? 0 : x)` kind of pattern looks weird.